src/NoFrills.js

```diff
@@ -8,17 +8,19 @@ class NoFrills {
 
   constructor() {
     this.stores = [];
+    this.flyers = [];
   }
 
   init() {
     return scraper.scrape()
-      .then(stores => {
-        this.stores = stores;
+      .then(data => {
+        this.stores = data.stores;
+        this.flyers = data.flyers;
       });
   }
 
   getStoreById(id) {
-    return this.stores.filter(store => store.id === id);
+    return this.stores.filter(store => store.id === id)[0];
```

Comment:
Is there a reason not to prefer:

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find